plugins/router/freebox

```diff
@@ -0,0 +1,316 @@
+#!/bin/bash
+# -*- sh -*-
+
+: << =cut
+
+=head1 NAME
+
+freebox - Plugin to monitor stats of a Freebox
+
+The ideal usage is also used as an updated warning limit.
+
+=head1 MAGIC MARKERS
+
+  #%# family=auto
+  #%# capabilities=autoconf
+
+=head1 CAVEAT
+
+Only tested on a Freebox v5 with an ADSL uplink.
+
+=head1 AUTHOR
+
+Olivier Mehani
+
+Copyright (C) 2019 Olivier Mehani <shtrom+munin@ssji.net>
+
+=head1 LICENSE
+
+SPDX-License-Identifier: GPL-3.0-or-later
+
+=cut
+
+# shellcheck disable=SC1090
+. "${MUNIN_LIBDIR}/plugins/plugin.sh"
+
+if [ "${MUNIN_DEBUG}" = 1 ]; then
+    set -x
+fi
+
+for CMD in curl iconv bc; do
```

Comment:
Please move this check to `autoconf`. Now it would cause the `autoconf` call to fail. This should not happen.